x-pack/plugins/session_view/public/components/process_tree_node/index.tsx

```diff
@@ -24,13 +24,16 @@ import { FormattedMessage } from '@kbn/i18n-react';
 import { Process } from '../../../common/types/process_tree';
 import { useStyles } from './styles';
 import { ProcessTreeAlerts } from '../process_tree_alerts';
-import { SessionLeaderButton, AlertButton, ChildrenProcessesButton } from './buttons';
+import moment from 'moment';
```

Comment:
Not sure we want to use moment. If I recall it's not a recommended lib to use anymore. Does kibana core have a util for date formating that is common across the app?